pydantic_factories/plugins/pytest_plugin.py

```diff
@@ -0,0 +1,90 @@
+import inspect
+import re
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Callable, Optional, Type, Union, overload
+
+import pytest
+
+from pydantic_factories.factory import ModelFactory
+
+if TYPE_CHECKING:
+    from _pytest.config import Config
+    from _pytest.scope import _ScopeName
+
+    Scope = Union["_ScopeName", Callable[[str, Config], "_ScopeName"]]
+
+
+def _get_fixture_name(name: str) -> str:
+    """from inflection.underscore."""
+    name = re.sub(r"([A-Z]+)([A-Z][a-z])", r"\1_\2", name)
```

Comment:
Please compile the regexes A variables outside the function and use inside it. It's better performance wise.

Reply:
Sure  👍